diff -cr shim-15/Make.defaults shim-15-patched/Make.defaults
*** shim-15/Make.defaults	2018-04-05 20:49:17.000000000 +0200
--- shim-15-patched/Make.defaults	2018-10-31 09:31:26.569637297 +0100
***************
*** 102,108 ****
  endif
  
  FORMAT		?= --target efi-app-$(ARCH)
! EFI_PATH	?= $(LIBDIR)/gnuefi
  
  MMSTEM		?= mm$(ARCH_SUFFIX)
  MMNAME		= $(MMSTEM).efi
--- 102,108 ----
  endif
  
  FORMAT		?= --target efi-app-$(ARCH)
! EFI_PATH	?= $(LIBDIR)
  
  MMSTEM		?= mm$(ARCH_SUFFIX)
  MMNAME		= $(MMSTEM).efi
